public/app/ui/components/search/results/search-results.html

```diff
@@ -36,8 +36,8 @@
                     <tbody>
                     <!-- ko foreach: { data: results, as: 'result' } -->
                     <tr class="result">
-                        <td data-bind="text: $index() + $parent.pager.start() + 1"></td>
-                        <!-- ko foreach: { data: $parent.fields, as: 'field' } -->
+                        <td data-bind="text: $index() + $parents[1].pager.start() + 1"></td>
```

Comment:
Passing the observable would be done in `search.html` line 57.   You need to pass the observable, not the value at the time of rendering.  The syntax is probably easier if you make a `pureComputed` in `SearchPage` that just does `return this.pager.start();`, then you can just pass that observable in the same way that the others are passed.